skt/runner.py

```diff
@@ -87,12 +87,12 @@ def __load_blacklist(self, filepath):
                     if line:
                         hostnames.append(line)
         except (IOError, OSError) as exc:
-            logging.error('Can\'t access {}!'.format(filepath))
+            logging.error('Can\'t access %s!', filepath)
```

Comment:
Can you please explain in the commit message *why* this change is needed?

Reply:
I add commit message

Reply:
The commit message doesn't explain *why* the change was needed. It quotes the documentation, but seeing only the message, noone would know what kind of change it describes. Again, mentioning that it resolves a pylint warning (and which one) would provide more useful information.